code/modules/projectiles/ammo_datums.dm

```diff
@@ -1167,14 +1167,35 @@ datum/ammo/bullet/revolver/tp44
 	name = "thermobaric rocket"
 	hud_state = "rocket_thermobaric"
 	flags_ammo_behavior = AMMO_ROCKET
-	damage = 200
+	damage = 40
+	penetration = 25
 	max_range = 30
+	sundering = 2
+	var/datum/effect_system/smoke_spread/smoke_system
+	var/danger_message = "<span class='danger'>The rocket explodes explodes into white gas!</span>"
```

Comment:
No autodoc

Reply:
Also danger_message is unused

Reply:
danger message works on hit

Reply:
I know it works im saying its useless to have it in a var